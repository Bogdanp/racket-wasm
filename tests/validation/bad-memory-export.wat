(module
 (export "m" (memory 0)))