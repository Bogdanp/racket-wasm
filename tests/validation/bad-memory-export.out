(#f . "index out of bounds (no items)\n  at: (export \"m\" (memidx 0))")